server/mock/datastore_mock.go

```diff
@@ -119,7 +119,7 @@ type ListLabelsFunc func(ctx context.Context, filter fleet.TeamFilter, opt fleet
 
 type LabelQueriesForHostFunc func(ctx context.Context, host *fleet.Host) (map[string]string, error)
 
-type RecordLabelQueryExecutionsFunc func(ctx context.Context, host *fleet.Host, results map[uint]*bool, t time.Time) error
+type RecordLabelQueryExecutionsFunc func(ctx context.Context, host *fleet.Host, results map[uint]*bool, t time.Time, deferredSaveHost bool) error
```

Comment:
The mock implementation had not been regenerated in the last PR, I caught the build errors when I did regenerate for the updated signature of GenerateHostStatusStatistics.

Reply:
How the heck did the test pass then :/